src/main/java/org/terasology/ligthandshadow/componentsystem/LASUtils.java

```diff
@@ -51,6 +51,8 @@
     public static final String WHITE_HEALTH_SKIN = "lightAndShadowResources:healthWhite";
     public static final String RED_HEALTH_SKIN = "lightAndShadowResources:healthRed";
     public static final String BLACK_HEALTH_SKIN = "lightAndShadowResources:healthBlack";
+    public static final String RED_MINIMAP_ICON = "lightAndShadowResources:heartsIcon";
```

Comment:
As pointed out in https://github.com/Terasology/LightAndShadowResources/pull/31#pullrequestreview-277364660 you might just use the icons that are already there:
```suggestion
    public static final String RED_MINIMAP_ICON = "lightAndShadowResources:heart";
```